horovod/mxnet/adapter.cc

```diff
@@ -0,0 +1,144 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+#if HAVE_CUDA
+#include "cuda.h"
+#endif
+
+#include <mxnet/base.h>
+
+#include "adapter.h"
+#include "cuda_util.h"
+#include "tensor_util.h"
+
+namespace horovod {
+namespace MX {
```

Comment:
@ctcyang Could you please merge my commit 86c89c72bfe4a170e18436ca5acefc66cd2c0568 so the namespace will be consistent with tensorflow and tf? Thanks!

Reply:
Done. Thanks!